articles/dms/howto-mysql-to-azure-mysql-powershell.md

```diff
@@ -0,0 +1,657 @@
+---
+title: "PowerShell: Run offline migration from MySQL database to Azure Database for MySQL" 
+titleSuffix: Azure Database Migration Service
+description: Learn to migrate an on-premise MySQL database to Azure Database for MySQL by using Azure Database Migration Service through Powershell script.
+services: dms
+author: "sumitgaurin"
+ms.author: "sgaur"
+manager: "arthiaga"
+ms.reviewer: "arthiaga"
+ms.service: dms
+ms.workload: data-services
+ms.custom: "seo-lt-2019"
+ms.topic: tutorial
+ms.date: 04/11/2021
+---
+
+# Migrate an on-premise MySQL database to Azure Database for MySQL by using Azure Database Migration Service through PowerShell script
+
+In this article, you migrate a MySQL database restored to an on-premises instance to Azure Database for MySQL by using the Offline migration capability of Azure Database Migration Service through Microsoft Azure PowerShell. You can migrate databases from a MySQL instance to Azure Database for MySQL instance by using the `Az.DataMigration` module in Microsoft Azure PowerShell. The article documents a collection of PowerShell scripts which can be executed in sequence to perform the offline migration of MySQL database to Azure.
```

Comment:
I think we should frame this content as it is not possible to migrate databases using the Az.DataMigration module, but that the capability will come online by the time of GA.  In the meantime, we have a sample PowerShell script we are providing "as-is" that uses the DMS Rest API (perhaps we can link to those docs?) and allows them to automate migration.  We should reiterate that this script will not be needed once official support is added in the Az.DataMigration module and Azure CLI.

Reply:
Added the following:
> [!NOTE]
> It is not possible to run complete database migration using the Az.DataMigration module, but that capability will come online by the time of GA. In the meantime, the sample PowerShell script is provided "as-is" that uses the [DMS Rest API](https://docs.microsoft.com/rest/api/datamigration/tasks/get) and allows you to automate migration. This script will be modified or deprecated, once official support is added in the Az.DataMigration module and Azure CLI.